<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Uploaded</class>
 <widget class="QDialog" name="Uploaded">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Starfiles - All done</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets/resources.qrc">
    <normaloff>:/images/images/logo/favicon.ico</normaloff>:/images/images/logo/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Uploaded {
	background-color:#ffffff;
}

QFrame {
	background-color:#f0f0f0;
}

QLabel {
	
	background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>410</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Your file has been successfully uploaded &lt;/span&gt;&lt;span style=&quot; font-weight:400; color:#000000;&quot;&gt;ðŸ¥³&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>411</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Titillium Web</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>File Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FileName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCopyFN">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets/resources.qrc">
         <normaloff>:/images/images/copy.svg</normaloff>:/images/images/copy.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>411</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Titillium Web</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Download Link :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DownloadLink">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCopyDL">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets/resources.qrc">
         <normaloff>:/images/images/copy.svg</normaloff>:/images/images/copy.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>160</y>
      <width>432</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="BtnOk">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>9</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>58</y>
      <width>411</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Titillium Web</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>File ID :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FileID">
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnCopyFID">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="assets/resources.qrc">
         <normaloff>:/images/images/copy.svg</normaloff>:/images/images/copy.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>16</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
